packages/govuk-frontend/src/govuk/components/header/template.test.js

```diff
@@ -301,11 +301,11 @@ describe('header', () => {
       expect($svg.html()).toContain('<title>GOV.UK</title>')
     })
 
-    it('uses the Tudor Crown if useTudorCrown is set', () => {
-      $ = render('header', examples['with Tudor crown'])
+    it("uses the St Edward's Crown if useTudorCrown is false", () => {
```

Comment:
Extremely pithy comment that we're using double quotes here when the pattern for test cases are single quotes